puppet/manifests/default.pp

```diff
@@ -2,8 +2,35 @@
 
   include oracle_java
 
+  # NOTE: This cert wrangling is only good for a single machine. We need some
+  # other mechanism to get our certs to the right place in a multi-node setup.
+  package { 'openssl': }
+  ->
+  file { '/var/docker-certs':
+    ensure => directory,
+  }
+  ->
+  openssl::certificate::x509 { 'docker-registry':
+    country => 'NT',
+    organization => 'seed-stack',
+    commonname => 'docker-registry.service.dc1.consul',
```

Comment:
Hopefully we can just make this `docker-registry.service.consul` and things should resolve. Consul should default to `dc1`.